### PR TITLE
Implement suggested refactoring.

### PR DESCRIPTION
**Pull request by AlekseyTs**

### CODE REVIEW

**Comment by AlekseyTs** [comment]
@333fred, @mavasani, @cston, @dotnet/roslyn-compiler, @dotnet/analyzer-ioperation Please review, very small change, no new tests.

**Review by mavasani — APPROVED** [review]

**Review by 333fred — APPROVED** [review]

**Comment by AlekseyTs** [comment]
@cston, @dotnet/roslyn-compiler Please review, very small change, no new tests. Need second review.

File: src/Compilers/Core/Portable/Operations/ControlFlowGraphBuilder.cs
Comment (jcouv) [inline src/Compilers/Core/Portable/Operations/ControlFlowGraphBuilder.cs]: Nit: empty line above

**Review by jcouv — APPROVED** [review]
LGTM Thanks

**Pull request merged**
